code/modules/bitrunning/server/util.dm

```diff
@@ -7,9 +7,10 @@
 	//BUBBER ADDITION BEGIN - This is a HORRIBLE HACK to stop the radio from blurting out on cargo channel for dauntless prisoners
 	//If the dauntless map is removed, remove this hack!
 	//If the server is moved into a different area for dauntless, update this hack!
-	var/turf/curr = get_turf(radio)
+	var/area/curr = get_area(src)
 	var/area/dauntless_prison = /area/ruin/space/has_grav/bubbers/dauntless/sec/prison
-	if(get_area_name(curr, TRUE) == dauntless_prison.name)
+	var/area/dauntless_space_prison = /area/ruin/space/has_grav/bubbers/dauntless_space/sec/prison
+	if(istype(curr, dauntless_prison) || istype(curr, dauntless_space_prison))
```

Comment:
We're probably better off checking either against `!is_station_level(src.z)` (This might not work on icebox and moon (?)), or against `/area/ruin/space/has_grav/bubbers/dauntless_space` and `/area/ruin/space/has_grav/bubbers/dauntless` directly if the above doesn't work. No need to use vars either, just check against the type directly in this case

Also could be nice to still have the machine at least make the statement locally with the say proc